pycqed/instrument_drivers/physical_instruments/NanotecSMI33.py

```diff
@@ -0,0 +1,768 @@
+import logging
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import Enum, Ints, Numbers
+
+"""
+Requires
+    pyVISA-py
+    PySerial
+"""
+
+log = logging.getLogger(__name__)
+
+
+class NanotecSMI33(VisaInstrument):
+    """
+    This is a driver for the Nanotec SMI33 Motor controller
+
+    Status: not tested
+
+    Only a subset of all features have been included
+    """
+    def __init__(self, name: str, address, controller_id: str, **kwargs) -> None:
+        """
+        Args:
+            name: The name of this instance
+            address: The address of the controller
+            controller_id: The id of the motor which can be either '*' or a number
+                between 1 and 254
+            kwargs: Additional keyword arguments
+        """
+        super().__init__(name, address, terminator='\r', visalib='@py',
```

Comment:
Does it only work with the visalib @py or should this remain something that the user can choose?
The second option would make more sense to me.

Reply:
No you are correct, it can also work with other visa backends. I have removed this from the latest commit and also tested on Qudev108.